packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -170,6 +164,11 @@ const ComboboxField = ({
     }
   }
 
+  if (customProps.placeholder) {
+    // eslint-disable-next-line no-param-reassign
+    customProps.placeholder = null;
```

Comment:
I think instead of assigning to null we can delete the prop 
```js
delete customProps.placeholder;
```

Reply:
Updated here https://github.com/cerner/terra-core/pull/3094/commits/d3d0df8877f5742edf80995be6aae5ba19b5ff6b